WeatherData.py

```diff
@@ -0,0 +1,99 @@
+class WeatherData:
+    """This class holds all the data of the weather.
+
+    Methods:
+        To add data in the structure, use method add_month_data
+        To get data use "get_column_data" method with month and year
+        To get the column name use method get_column_name_from_alias.
+            Valid aliases are
+                max_temperature
+                min_temperature
+                max_humidity
+                min_humidity
+                mean_humidity
```

Comment:
add an example of an object of this class, and what is the structure of the data it holds!